content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+
+Kamu dapat menggunakan `kubeadm reset` pada _host_ _control plane_ untuk memicu pembersihan
+best-effort.
+
+Lihat dokumentasi referensi [`kubeadm reset`](/docs/reference/setup-tools/kubeadm/kubeadm-reset/)
+untuk informasi lebih lanjut mengenai sub-perintah ini dan
+opsinya.
+
+{{% /capture %}}
+
+{{% capture discussion %}}
+
+## Selanjutnya {#selanjutnya}
+
+* Pastikan klaster berjalan dengan benar menggunakan [Sonobuoy](https://github.com/heptio/sonobuoy)
+* <a id="lifecycle" />Lihat [Memperbaharui klaster kubeadm](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
+  untuk detail mengenai pembaruan klaster menggunakan `kubeadm`.
+* Pelajari penggunaan `kubeadm` lebih lanjut pada [dokumentasi referensi kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm)
+* Pelajari lebih lanjut mengenai [konsep-konsep](/docs/concepts/) Kubernetes dan [`kubectl`](/docs/user-guide/kubectl-overview/).
+* Lihat halaman [Cluster Networking](/docs/concepts/cluster-administration/networking/) untuk daftar
+_add-on_ jaringan Pod yang lebih banyak.
+* <a id="other-addons" />Lihat [daftar _add-on_](/docs/concepts/cluster-administration/addons/) untuk
+  mengeksplor _add-on_ lainnya, termasuk perangkat untuk _logging_, _monitoring_, network policy, visualisasi &amp;
+  pengendalian klaster Kubernetes.
+* Atur bagaimana klaster mengelola _log_ untuk peristiwa-peristiwa klaster dan dari
+  aplikasi-aplikasi yang berjalan pada Pod.
+  Lihat [Arsitektur Logging](/docs/concepts/cluster-administration/logging/) untuk
+  gambaran umum tentang hal-hal yang terlibat.
+
+### Umpan balik {#feedback}
+
+* Untuk _bug_, kunjungi [kubeadm GitHub issue tracker](https://github.com/kubernetes/kubeadm/issues)
+* Untuk dukungan, kunjungi kanal Slack
+  [#kubeadm](https://kubernetes.slack.com/messages/kubeadm/)
+* Kanal Slack umum pengembangan SIG Cluster Lifecycle:
+  [#sig-cluster-lifecycle](https://kubernetes.slack.com/messages/sig-cluster-lifecycle/)
+* SIG Cluster Lifecycle [SIG information](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle#readme)
+* Milis SIG Cluster Lifecycle:
+  [kubernetes-sig-cluster-lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
+
+## Kebijakan versi _skew_ {#version-skew-policy}
+
+`kubeadm` versi v{{< skew latestVersion >}} dapat menyebarkan klaster dengan _control plane_ versi v{{< skew latestVersion >}} atau v{{< skew prevMinorVersion >}}.
+`kubeadm` v{{< skew latestVersion >}} juga dapat memperbarui klaster yang dibuat dengan kubeadm v{{< skew prevMinorVersion >}}.
+
+Karena kita tidak dapat melihat masa depan, kubeadm CLI v{{< skew latestVersion >}} mungkin atau tidak mungkin dapat menyebarkan klaster v{{< skew nextMinorVersion >}}.
+
+Sumber daya ini menyediakan informasi lebih lanjut mengenai versi _skew_ yang didukung antara kubelet dan _control plane_, serta komponen Kubernetes lainnya:
+
+* Kubernetes [version and version-skew policy](/docs/setup/release/version-skew-policy/)
```

Comment:
```suggestion
* [Kebijakan versi and version-skew Kubernetes](/docs/setup/release/version-skew-policy/)
```

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc